app/controllers/generic_files_controller.rb

```diff
@@ -9,11 +9,29 @@ class GenericFilesController < ApplicationController
   self.edit_form_class = FileEditForm
 
   def update_metadata
+    params["generic_file"]["nested_geo_bbox_attributes"].each do |box, value|
+      if [value["label"], value["bbox_lat_north"], value["bbox_lon_west"], value["bbox_lat_south"], value["bbox_lon_east"]].none? { |f| f.empty? }
+        bbox = [value["bbox_lat_north"], value["bbox_lon_west"], value["bbox_lat_south"], value["bbox_lon_east"]]
+        value["bbox"] = bbox.to_s
+      end
+    end
     file_attributes = edit_form_class.model_attributes(params[:generic_file])
     updated_attributes = AttributeURIConverter.new(file_attributes).convert_attributes
     actor.update_metadata(updated_attributes, params[:visibility])
   end
 
+  # routed to /files/:id/edit
+  def edit
+    @generic_file.nested_geo_bbox.each do |box|
+      box_array = box.bbox.to_a.first.split(",").map { |s| s.tr('[]" ','').to_s }
```

Comment:
This line seems like it is worthy of a slight refactor and comment so that our future self will understand what the intent it.

`# bbox is stored as a string array of lat/long string arrays like: '["121.1", "121.2", "44.1", "44.2"]', however only one array of lat/long array is stored, so the first will need to be converted to simple array of strings like: ["121.1","121.2","44.1","44.2"]
box_array = box.bbox.to_a.first.tr('[]" ','').split(',')
`